text/XXXX-delete_and_drop.md

```diff
@@ -0,0 +1,225 @@
+- Feature Name: delete and drop
+- Start Date: 2020-05-02
+- RFC PR: [#53](https://github.com/qri-io/rfcs/pulls/53)
+- Issue: <!-- (leave this empty) -->
+
+# Summary
+[summary]: #summary
+
+Rename the remove command to `delete`, and clarify it's purpose is for deleting versions from a dataset history. Create a new command `drop` for removing datasets from remotes.
+
+# Motivation
+[motivation]: #motivation
+
+The _user experience (UX)_ of removing data from Qri needs some work. We're seeing a number of users get into states they feel they cannot recover from, and rather than use tools like `qri remove`, they're deleting repositories entirely in an effort to start over. 
+
+A better UX for removing data from qri should clearly answer two common "uh oh" scenarios a user is likely to encounter:
+
+1. "I made a mistake while saving"
+2. "I want to undo a push"
+
+The first is an example of removing data from history. The second is an example of removing data from the network. In both scenarios the user is a data author, and a good answer to these questions should present well-documented, easy-to-find commands that when executed leave the user feeling that they're in a clean state. 
+
+A well-documented desctrive command should be clearly labeled as the "opposite" of a constructive command and present warnings _before_ execution. Once executed, commands remove data should show as few caveat/errors/warnings as possible, or at least connect caveats to a clear next course of action. A feeling of "clean state" comes when vommands like `qri log`, `qri list` and `qri status` match user expectations after running a destructive command.
+
+To improve our "undo UX", this RFC proposes three changes to qri CLI:
+1. rename `remove` to `delete`, clarify its text to talk about history instead of just storage
+2. Add an `amend` flag on the `save` command that _replaces_ the HEAD commit
+3. Add a new top-level command: `drop`, for dropping datasets you don't own
```

Comment:
I'm not a fan of `drop` at all. First off, the language around "datasets you own" is tricky to explain to users. We often talk about how, before you can do certain operations, you have to get the dataset onto your machine (using what used to be `add` and what will soon become `pull`). It's confusing to talk about having a dataset in your collection vs owning it, when the difference should be transparent. Like, with a file I've downloaded versus authored, or a git repo I've cloned vs init'ed, in either case is just a thing sitting on my machine. Maybe I'm wrong on this, but all I can imagine is a user thinking "I see this dataset when I list (or see it in my Collections on Desktop) and just want to get rid of it", there should only be a single command to make it stop showing up.

This feels like reinventing the "fsi=true" problem, where qri is operating on "mode a" and "mode b", and you have to use operation A or operation B. Qri knows which operation is for each mode, but if you get it wrong there's just an error, even though qri could instead treat them as the same thing.

I understand the fear around "oops I meant to remove something I didn't own, or from a remote instead of my machine", but I would much rather have an undo escape hatch instead of a confusing UI. For instance, git has the dangerous `git reset --hard old-commit` which can be undone with `git reset --hard ORIG_HEAD` because its immutable objects exist until gc is run. We could have a similar `git restore ORIG_REF` or something thanks to logbook.

For removing from a remote I would much rather see `git remove me/oops_i_pushed_it --from-remote registry`.